### PR TITLE
Windows compile

### PR DESCRIPTION
**Pull request by SomethingGettingWrong**

### CODE REVIEW

File: Graft Network Windows Compile.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+```
+or for the community edition
+```
+git clone https://github.com/graft-community/GraftNetwork.git
+```
+
+Once its done type (this will put you in the current directory of
+GraftNetwork)
+
+`Cd GraftNetwork`
```
Comment (jagerman) [inline Graft Network Windows Compile.md]: This probably needs to be a lower-case `c`

File: Graft Network Windows Compile.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+
+# GRAFT COMMUNITY
+### Windows Compile Instructions For GRAFT NETWORK
+##### By Patrick Durbin @SomethingGettingWrong
+![GClogo](https://github.com/Patrickdurbin/docs/blob/Windows-Compile/windows-compile-images/GClogo.jpg)
+
+First head on over to the msys64 website.[MSYS2.ORG](https://www.msys2.org/)
```
Comment (jagerman) [inline Graft Network Windows Compile.md]: This looks a bit odd without a space between the . and the MSYS2.ORG link

File: Graft Network Windows Compile.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+![git](https://github.com/Patrickdurbin/docs/blob/Windows-Compile/windows-compile-images/git.jpg)
+
+Exit out of msys
+
+Please browse back to the C:\msys64 and look for mingw
+
+#### Run mingw
+```
+git clone https://github.com/graft-project/GraftNetwork.git
```
Comment (jagerman) [inline Graft Network Windows Compile.md]: You can skip the `git submodule update` section that follows by telling git to do a clone + submodule clone all at once:
```
git clone --recurse-submodules https://github.com/graft-project/GraftNetwork.git
```
and then just remove the paragraph below about the `submodule update`

File: Graft Network Windows Compile.md (hunk 1 above)
Comment (SomethingGettingWrong) [reply Graft Network Windows Compile.md]: Yes will change.

File: Graft Network Windows Compile.md (hunk 2 above)
Comment (SomethingGettingWrong) [reply Graft Network Windows Compile.md]: I can commit to adding a space I suppose

File: Graft Network Windows Compile.md (hunk 3 above)
Comment (SomethingGettingWrong) [reply Graft Network Windows Compile.md]: Alright will do.

File: Graft Network Windows Compile.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Please browse back to the C:\msys64 and look for mingw
+
+#### Run mingw
+```
+git clone --recurse-submodules https://github.com/graft-project/GraftNetwork.git
+```
+or for the community edition
+```
+git clone --recurse-submodules https://github.com/graft-project/GraftNetwork.git
```
Comment (jagerman) [inline Graft Network Windows Compile.md]: `graft-project` -> `graft-community`.

Comment (SomethingGettingWrong) [reply Graft Network Windows Compile.md]: fixed!

**Pull request merged**